src/box/iproto.cc

```diff
@@ -2639,28 +2701,35 @@ static void
 tx_process_override(struct cmsg *m)
 {
 	struct iproto_msg *msg = tx_accept_msg(m);
-	mh_int_t k = mh_i32ptr_find(tx_req_handlers, msg->header.type, NULL);
+	mh_int_t k;
+	struct mh_i32ptr_node_t *node;
+	struct iproto_req_handler *handler;
+	const char *header;
+	const char *header_end;
+	const char *body = "\x80"; /* Empty MsgPack map encoding. */
+	const char *body_end = body + 1;
+
+	if (iproto_shutdown_started) {
+		diag_set(FiberIsCancelled);
+		goto error;
+	}
```

Comment:
I don't understand why you need to handle this case explicitly.

Reply:
Other functions processing requests use `tx_check_msg` which has this check. This handler does not use the function so we have to add it explicitly. Anyway we changed the way we cancel requests which reside in queues and do not need this check anymore.